### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#296] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-10-28 110529](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/e8d6ab3f-9754-46e2-bc9b-338e36c0eaf1)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/a96f64cd-5a69-4898-bf94-667177f03891)

**Comment by Frisbon** [comment]
Soluzione diversa lol spero vada bene
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/87118e2a-51ec-4c1f-8c6e-6f855545c7c4)
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/d7af2e46-64b2-4ce9-a2ca-1913cfc4f0b5)
De quello che ha detto gpt non cho capitonucazz

**Reply by daromacarlo** [reply]
@Frisbon Ti sconsiglio di usare gpt per questi esercizi, riesce a creare frasi di senso compiuto ma niente di più.
